plutarch-test/plutarch-test.cabal

```diff
@@ -94,20 +95,24 @@ common deps
 
 library
   import:          c, deps
-  hs-source-dirs:  src
+  hs-source-dirs:  common
   exposed-modules: Plutarch.Test
   other-modules:
     Plutarch.Test.Benchmark
     Plutarch.Test.Golden
     Plutarch.Test.ListSyntax
 
+  if flag(development)
+    cpp-options: -DDevelopment
+
 executable plutarch-test
   import:         c, deps
   main-is:        Main.hs
-  hs-source-dirs: src
+  hs-source-dirs: plutarch-base plutarch-extra common ./.
```

Comment:
I'm curious, why `plutarch-base` instead of `plutarch`? The cabal library name is still `plutarch` AFAICT, unless you changed that too.

Reply:
To allow for better distinction, I can change that as well if it is unwanted.